PocketKit/Sources/Sync/PocketSource.swift

```diff
@@ -264,10 +264,6 @@ public class PocketSource: Source {
 // MARK: - Saves/Archive items
 extension PocketSource {
     public func refreshSaves(completion: (() -> Void)? = nil) {
-        if lastRefresh.lastRefreshSaves == nil {
```

Comment:
This function could be called mutltiple times, moving this send state to the Operation is safer because the operation can only be called one at a time.